whois.json

```diff
@@ -11,14 +11,14 @@
     "product_vendor": "Generic",
     "product_name": "Whois",
     "product_version_regex": ".*",
-    "min_phantom_version": "4.10.0.40961",
+    "min_phantom_version": "5.0.0",
```

Comment:
please change min phantom to 5.1.0

Reply:
ok, done. sorry didn't see we changed min to 5.1 yet.